pkg/cgroups/cgrouppath.go

```diff
@@ -83,3 +88,14 @@ func init() {
 	flag.StringVar(&KubeletRoot, "kubelet-cgroup-root", KubeletRoot,
 		"--cgroup-root options the kubelet is running with")
 }
+
+func DetectSystemCgroupVersion() int {
```

Comment:
I'd suggest to do smth like this instead, possibly import/copy it directly
https://github.com/containerd/cgroups/blob/e8802a182774b8e25cc65b5c374389338cc8e6c3/utils.go#L53